packages/ui/src/components/va-carousel/VaCarousel.vue

```diff
@@ -86,7 +86,7 @@
               v-bind="vaImageProps"
               :src="isObjectSlides ? item.src : item"
               :alt="isObjectSlides ? item.alt : ''"
-              :draggable="false"
+              not-draggable
```

Comment:
In other places we don't use `not`, so my concern is that this place breaks a convention.